lib/action_subscriber/base.rb

```diff
@@ -80,7 +80,8 @@ def _at_least_once_filter
       yield
       acknowledge
     rescue => error
-      reject
+      ::ActionSubscriber::MessageRetry.redeliver_message_with_backoff(env)
```

Comment:
I wanted to point out here, that we are publishing the message retries with `publisher_confirms` on, so this pattern here means that if our code crashes between publishing the retry and when we acknowledge we could end up duplicating the message.

I'm not very worried about this because if people are turning on `at_least_once!` I assume they are okay with making their subscriber idempotent, but just wanted to point it out in case someone thinks of something I'm missing